sorc/chgres_cube.fd/model_grid.F90

```diff
@@ -609,11 +609,14 @@ subroutine define_input_grid_gfs_grib2(localpet, npets)
 
  implicit none
 
+ include "mpif.h"
```

Comment:
Consider using mpi module (`use mpi`)

Reply:
I used the same syntax that is used in other parts of the code, in particular in the changes that I cherry-picked from commit 790a7603e2d899c5db2c4e63e22cc0ee62aaaa60 (feature/mpi_abort Fix mpi_abort calls in chgres_cube and
sfc_climo_gen). I would suggest to change to `use mpi` in the develop branch, and consider bringing the bugfix for reading grib2 files from this PR to develop, too.

Reply:
I will open an issue to replace the include 'mpif.h' with 'use mpi', and to include the bug fix.  @climbfuji why not use @DusanJovic-NOAA suggestion in the release branch as well?

Reply:
> I will open an issue to replace the include 'mpif.h' with 'use mpi', and to include the bug fix. @climbfuji why not use @DusanJovic-NOAA suggestion in the release branch as well?

Alright, I'll make the change in all places I can find. But it means yet another round of tests on a range of systems before we can merge, retag, roll out the libraries and do more testing.

Reply:
> > I will open an issue to replace the include 'mpif.h' with 'use mpi', and to include the bug fix. @climbfuji why not use @DusanJovic-NOAA suggestion in the release branch as well?
> 
> Alright, I'll make the change in all places I can find. But it means yet another round of tests on a range of systems before we can merge, retag, roll out the libraries and do more testing.

@climbfuji if it is more trouble than it is worth, then leave things as they are.